### DISCUSSION TITLE
Automount USB when no user is logged in desktop

### DISCUSSION THREAD

**Discussion by fawzibr** [coldfix/udiskie#268] [Q&A]
This is not really a udiskie question more like a udisks2 one and after googling around with no answers decided to try here.

Is it possible to have udisks2 mount a usb device as soon as it is connected (even if no user is logged in)? 

I have a script that backups stuff to usb. I cant use fstab because the disks can change to I though of udev. Made a rule that mounts the usb. Works as long as I don't log in on the desktop. 

I thought that maybe it was possible with d-bus messages but have no idea how that would work. I can see with dbus-monitor that the messages come in when usb disks are connected like:

```
11:14:40.458: /org/freedesktop/UDisks2/block_devices/sdb: org.freedesktop.UDisks2.PartitionTable: Properties Changed
  Partitions:           /org/freedesktop/UDisks2/block_devices/sdb1
```
I just want to mount at that moment and somehow stop udisks2 from killing those mounts when an user logs in in the desktop. Any ideas/recommendations? Can anyone here point me in a direction?

**Comment by coldfix** [comment]
Hi, sorry for not answering. I didn't see your question in the discussions tab.

Yes, I believe that should be possible. You could try to e.g. run udiskie as systemd service to let it automount even when you haven't logged in.
